app/controllers/content_items_controller.rb

```diff
@@ -3,20 +3,17 @@
 
 class ContentItemsController < ApplicationController
   include Slimmer::Headers
+  include Slimmer::Template
   rescue_from GdsApi::HTTPForbidden, with: :error_403
 
   def show
-    # The Slimmer middleware is responsible for intercepting the response and
-    # wrapping it with the GOV.UK header and footer. We want to use our own
-    # 'report a problem' pattern, so opt out of the bundled one.
-    set_slimmer_headers(
-      report_a_problem: "false"
-    )
+    slimmer_template :without_footer_links
```

Comment:
Do we want to disable footers across the service manual, in addition to the service toolkit?